cohorts/cohort.py

```diff
@@ -326,20 +322,27 @@ def is_lambda(func):
         if type(on) == dict:
             cols = []
             for key, value in on.iteritems():
-                col, df = apply_func(on=value, col=key, df=df)
+                if type(value) == str:
+                    df[key] = df[value]
+                    col = key
+                elif type(value) == FunctionType:
+                    col, df = apply_func(on=value, col=key, df=df)
                 cols.append(col)
         if type(on) == list:
```

Comment:
> don't use elif is wrong

Yea not wrong, just wanted to make sure I understood it right and only one was possible.